Felienne/hedy
# Quiz tab

- Closes #733
- Moves quizzes to a special adventure tab called "End/Eind"

Implementation details:
- Because quizzes are implemented as standalone pages, we load them inside an iframe. The quiz templates have been modified, but not extensively, to hide the navigation bar (which is no longer necessary in the context of an iframe).
- Before this change, the button to go to a quiz was shown depending on the available quizzes in `coursedata/quiz`. After this change, this will happen depending on whether there's an `end` adventure for a given level. It is important to set and maintain a 1:1 relationship between these `end` entries in adventures and the quiz content.

**How to test**:
- Quizzes are enabled for levels 1 & 2. The texts are all in Dutch since they haven't been translated (and the quiz code doesn't support internationalization yet).
- By going to the "End/Eind" tab, the whole quiz flow (start quiz button, start quiz screen, questions, quiz end) should be available.
- When looking at a quiz, two sections should be gone: 1) the inputs & button for saving/sharing a program; 2) the editor and output boxes.
